core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/arrow/ArrowNltkSentimentOperator.java

```diff
@@ -59,6 +48,11 @@
                     new Field("text", FieldType.nullable(new ArrowType.Utf8()), null))
             );
 
+    // Flight related
+    private final static Location lc = new Location(URI.create("grpc+tcp://localhost:5005"));
```

Comment:
Rename `lc` to `location`. In general, try not to use abbreviation for variable naming, it is not worth it.